.github/workflows/triage.yml

```
@@ -165,7 +165,7 @@ jobs:
               keep_if_no_match: true
 
             - label: CI-linux-self-hosted
-              path: Formula/(gdbm|go|openssl@1.1|python@3.10).rb
+              path: Formula/(aws-sdk-cpp|gdbm|llvm|openssl@1.1|python@3.10|qt(@5)?).rb
```

```suggestion
              path: Formula/(gdbm|llvm|openssl@1.1|python@3.10|qt(@5)?).rb
```

Looks like a recent `aws-sdk-cpp` run (#112690) didn't hit OOM issue.

---

* `go` still skips most dependents so previous runs took only ~30min.
* `llvm` always goes above 6 hours
* `qt` and `qt@5` - can't remember reason but either/both 6 hour limit or/and disk space.

---

libxcrypt probably can be tested non-recursively like OpenSSL is. An ABI breakage in `libxcrypt` would be significant and will be picked up from a first-level run of dependencies anyway.